### PR TITLE
Add role prop to breadcrumb item

### PR DESCRIPTION
**Pull request by erichdev**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ yarn change`

#### Description of changes

Add optional `role` prop to `IBreadcrumbItem` that gets passed to the `<Link>`. Breadcrumb items render as a button by default, or as a link if an `href` is passed in. This prop will allow consumers to give breadcrumb items a role of `link`, for example, while still rendering it as a button.

### CODE REVIEW

File: packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md [hunk 1]
```diff
@@ -1768,6 +1768,7 @@ export interface IBreadcrumbItem {
     isCurrentItem?: boolean;
     key: string;
     onClick?: (ev?: React.MouseEvent<HTMLElement>, item?: IBreadcrumbItem) => void;
+    role?: string;
```
Comment (erichdev) [inline packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md:1771]: Looking at this now, I think maybe it'd be better to add this role to `IBreadcrumbProps`? I don't know if there'd ever be a scenario where items within the same breadcrumb would have different roles

Comment (xugao) [reply packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md:1771]: i do think it's more suitable to add it here, as what you have now

**Comment by xugao** [comment]
could you pls make the same fix in `react-next` package? thanks!
https://github.com/microsoft/fluentui/blob/master/packages/react-next/src/components/Breadcrumb/Breadcrumb.types.ts#L110
https://github.com/microsoft/fluentui/blob/master/packages/react-next/src/components/Breadcrumb/Breadcrumb.base.tsx#L258

**Comment by erichdev** [comment]
> could you pls make the same fix in `react-next` package? thanks!
> https://github.com/microsoft/fluentui/blob/master/packages/react-next/src/components/Breadcrumb/Breadcrumb.types.ts#L110
> https://github.com/microsoft/fluentui/blob/master/packages/react-next/src/components/Breadcrumb/Breadcrumb.base.tsx#L258

I've pushed changes to react-next. Thanks!

**Review by xugao — APPROVED** [review]
LGTM, thanks!

**Pull request merged**